generic/criu.py.org

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
```

Comment:
why is the test case file name generic/criu.py.org ?

Reply:
Yes, it was some temporary file I have kept in my local copy. @nasastry already commented on this. I need to re-send PR with the changes (review comments incorporated).